### PR TITLE
ci(devenv): fully build SOLVCON with devenv

### PR DESCRIPTION
**Pull request by tai271828**
The latest devenv begins to support SOLVCON build fully
https://github.com/solvcon/devenv/pull/91 . It means that conda is not
necessary anymore. Thus we can run our CI with devenv fully without
conda.

### CODE REVIEW

**Comment by tai271828** [comment]
Since the CI for now will take much longer time if we build many packages with `devenv`. We may talk about the CI trigger policies of `SOLVCON` and `devenv`. For example, after landing this pull request, the next step could be :

1. trigger `devenv launch solvcon` CI as daily build and pull request
2. trigger unittest CI for each code push only
3. trigger `SOLVCON` CI with `devenv launch solvcon` as daily build and pull request
4. trigger smaller test cases for each code push only

...vise versa.

**Review by yungyuc — CHANGES_REQUESTED** [review]
I think we may not merge this as is, because a build takes almost an hour to finish.  Please keep a per-checkin/merge CI run to be complete in 3-5 minutes.

The full devenv build should be run only weekly.  It should use a separate run.

**Pull request closed without merging**
